spec/spec.md

```diff
@@ -605,7 +605,7 @@ Below is a non-normative example of a DID Document that includes a serviceEndpoi
 {
   "credentialStatus": {
     "id": "Qmdfr32sdf32546...",
-    "type": "StatusList2021",
+    "type": "RevocationList2021Status",
```

Comment:
Was this just us being out of alignment with the spec?

Reply:
Yep for Profile v0.1 since we're pinning the tagged v0.0.1 StatusList spec.

When we start preparing the Profile for v1.0 then all the StatusList stuff can be changed back to latest version again.